README.md

```diff
@@ -14,18 +26,17 @@ the hoods to run benchmarks on JVM.
 
 # Requirements
 
-Gradle 6.0 or newer
+Gradle 6.8 or newer
 
 Kotlin 1.4.30 or newer
 
 # Gradle plugin
 
-Add repository in `settings.gradle` to enable bintray repository for plugin lookup
+Add repository in `settings.gradle` to enable Gradle Plugin Portal for plugin lookup
```

Comment:
Yes. This is not required anymore.
Gradle will look if it finds the plugin in its "build in" tool. Otherwise in the gradlePluginPortal... So this section can be removed.

Reply:
Removed